variables.tf

```diff
@@ -190,7 +190,7 @@ variable "external_secrets_secrets_manager_arns" {
 variable "create_default_irsa" {
   description = "Create default IRSA for service account"
   type        = bool
-  default     = false
+  default     = true
```

Comment:
why we are creating irsa.? we have move that to parent module right, with s3 permissions

Reply:
the module was creating irsa earlier, so can release it as a non breaking version, when I'm calling, I will set it explicitly false to not create irsa.